src/i18n/en.json

```diff
@@ -593,9 +593,10 @@
     "status.loading": "Loading{optionalText}...",
     "supporting.documents.title": "extension support",
     "supporting.documents.subtitle": "Supporting guides and documentation",
-    "supporting.documents.link1": "How Lazy Consensus works",
-    "supporting.documents.link2": "Understanding Payments",
-    "supporting.documents.link3": "How Motions and Disputes work",
+    "supporting.documents.votingReputation.lazyConsensusLink": "Understanding Lazy Consensus",
```

Comment:
I would vote for putting these closer to where they are used. This file is more for general translations that are used in more than one place.